### PR TITLE
fix(flake): add missing meta section to flake

### PR DESCRIPTION
**Pull request by ch4og**
## Description of change
This adds meta section which is needed for example to show description of a package

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -22,6 +22,11 @@
               "--skip=test_extract_strings"
               "--skip=test_init"
             ];
+            meta = with pkgs.lib; {
+              description = "Analyze ELF binaries like a boss";
+              homepage = "https://binsider.dev/";
+              license = licenses.mit;
```
Comment (orhun) [inline flake.nix:28]: Can we also add apache-2.0 here?

Comment (ch4og) [reply flake.nix]: Done

**Review by orhun — APPROVED** [review]
Thanks!

**Comment by orhun** [comment]
btw, do you know if there are any GitHub actions (or can we hack something together) to test the Nix builds in CI?

**Comment by ch4og** [comment]
Easiest approach is installing nix package manager onto action

I'll look into it now and will create PR / give you feedback

**Comment by orhun** [comment]
Ah, for sure, please go ahead and create a draft PR. I can ping some Nix people for further help :)

Thanks for your effort on this! ❤️

**Pull request merged**